pkg/controller/controller.go

```diff
@@ -92,8 +92,8 @@ type ErrorDetails struct {
 
 const iapHeaderName = "x-goog-authenticated-user-email"
 
-func New(h *renderer.Renderer, p *justification.Processor, allowlist []string) (*Controller, error) {
-	categories, err := getCatagoriesDisplayData(p.Validators())
+func New(h *renderer.Renderer, p *justification.Processor, allowlist []string, ctx context.Context) (*Controller, error) {
```

Comment:
Context should always be the _first_ parameter to a function.

```suggestion
func New(ctx context.Context, h *renderer.Renderer, p *justification.Processor, allowlist []string) (*Controller, error) {
```

Reply:
Updated in the latest version. Thank you.